From: Arnaud Rebillout <arnaudr@kali.org>
Date: Wed, 21 May 2025 22:27:20 +0700
Subject: Don't install executable as root

So that the package can build with `Rules-Requires-Root: no`.
---
 rcracki_mt/Makefile      | 2 +-
 rcracki_mt/Makefile.cuda | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/rcracki_mt/Makefile b/rcracki_mt/Makefile
index 3ee4c87..0cf467a 100644
--- a/rcracki_mt/Makefile
+++ b/rcracki_mt/Makefile
@@ -126,7 +126,7 @@ gprofall: rcracki_mt
 
 install:
 	install -d $(BIN)
-	install --group=root --owner=root --mode=755 rcracki_mt $(BIN)
+	install --mode=755 rcracki_mt $(BIN)
 
 release: clean
 	mkdir $(RELEASE_DIR)
diff --git a/rcracki_mt/Makefile.cuda b/rcracki_mt/Makefile.cuda
index 85fd8ed..da3a45a 100644
--- a/rcracki_mt/Makefile.cuda
+++ b/rcracki_mt/Makefile.cuda
@@ -122,7 +122,7 @@ gprofall: rcracki_mt
 
 install:
 	install -d $(BIN)
-	install --group=root --owner=root --mode=755 rcracki_mt $(BIN)
+	install --mode=755 rcracki_mt $(BIN)
 
 release: clean
 	mkdir $(RELEASE_DIR)
